integration/src/test/java/org/apache/iotdb/session/IoTDBSessionSimpleIT.java

```diff
@@ -782,6 +782,42 @@ public void testInsertOneDeviceRecordsWithIncorrectOrder()
     session.close();
   }
 
+  @Test
+  public void testInsertStringRecordsOfOneDeviceWithOrder()
+      throws IoTDBConnectionException, StatementExecutionException {
+    session = new Session("127.0.0.1", 6667, "root", "root");
+    session.open();
+    List<Long> times = new ArrayList<>();
+    List<List<String>> measurements = new ArrayList<>();
+    List<List<String>> values = new ArrayList<>();
+
+    addStringLine(times, measurements, values, 1L, "s4", "s5", "5.0", "true");
+    addStringLine(times, measurements, values, 2L, "s2", "s3", "3", "4");
+    addStringLine(times, measurements, values, 3L, "s1", "s2", "false", "6");
+
+    session.insertStringRecordsOfOneDevice("root.sg.d1", times, measurements, values, true);
```

Comment:
You should also test a scenario that isAligned = false.

Reply:
I've added the aligned methods' unit tests.